### PR TITLE
Create remote access

### PR DESCRIPTION
**Pull request by Sigabort42**
add method create_remote_access

### CODE REVIEW

File: examples/create_remote_access.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+INFO = {"type": "CbwRam::RemoteAccess::Ssh::WithPassword",
+        "address": "",
+        "port": "22",
+        "login": "loginssh",
+        "password": "passwordssh",
+        "key": "",# precises the key of the connection
+        "node": "master"# precises the Cyberwatch source of the connection,
+        }
+
+CLIENT.create_remote_access(INFO)
```
Comment (jinxka) [inline examples/create_remote_access.py]: add new line at the end of file

File: spec/test_cbw_api.py [hunk 2]
```diff
@@ -108,3 +108,37 @@ def test_remote_accesses():
             assert isinstance(response, list) is True
             for remote in response:
                 assert isinstance(remote, CBWRemoteAccess) is True
+
+    @staticmethod
+    def test_create_remote_access():
+        """Tests for method remote_accesses"""
```
Comment (jinxka) [inline spec/test_cbw_api.py]: bad comment

File: spec/test_cbw_api.py [hunk 3]
```diff
@@ -108,3 +108,37 @@ def test_remote_accesses():
...
+                "key": "",  # precises the key of the connection
+                "node": "master"  # precises the Cyberwatch source of the connection,
+                }
+
+        with vcr.use_cassette('spec/fixtures/vcr_cassettes/create_remote_access.yaml'):
+            response = client.create_remote_access(info)
+
+            assert response is True
+
+        info = {"type": "CbwRam::RemoteAccess::Ssh::WithPassword",
```
Comment (jinxka) [inline spec/test_cbw_api.py]: if the address is the only field changing, you can just write `info["address"] = ""`

File: examples/create_remote_access.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+from cbw_api_toolbox.cbw_api import CBWApi
+
+API_KEY = 'x1WJYbzCckw88DW7DQae31MAcsvikqzuWVjHjWO8njg='
+SECRET_KEY = 'qPa780IvttvAEwKRlNzgIgYzlYx2ak0vH2c9MXUEfeg6Qq7UpasdSAg2EGNbmbN3yuuTwhptPinVjIkGBMJ0CQ=='
+API_URL = 'http://10.10.1.111'
+
+CLIENT = CBWApi(API_URL, API_KEY, SECRET_KEY)
+
+INFO = {"type": "CbwRam::RemoteAccess::Ssh::WithPassword",
+        "address": "",
```
Comment (jinxka) [inline examples/create_remote_access.py]: "address": "XXX.XXX.XXX.XXX",

File: examples/create_remote_access.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+"""Create connexion agentless"""
+
+from cbw_api_toolbox.cbw_api import CBWApi
+
+API_KEY = 'x1WJYbzCckw88DW7DQae31MAcsvikqzuWVjHjWO8njg='
+SECRET_KEY = 'qPa780IvttvAEwKRlNzgIgYzlYx2ak0vH2c9MXUEfeg6Qq7UpasdSAg2EGNbmbN3yuuTwhptPinVjIkGBMJ0CQ=='
```
Comment (jinxka) [inline examples/create_remote_access.py]: remove key

File: examples/create_remote_access.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+API_URL = 'http://10.10.1.111'
```
Comment (jinxka) [inline examples/create_remote_access.py]: remove url

File: cbw_api_toolbox/cbw_api.py [hunk 7]
```diff
@@ -106,3 +106,21 @@ def remote_accesses(self):
         """GET request to /api/v2/remote_accesses to get all servers"""
         response = self._request("GET", [ROUTE_REMOTE_ACCESSES])
         return CBWParser().parse_response(CBWRemoteAccess, response)
+
+    def create_remote_access(self, info):
+        """"POST request to /api/v2/remote_accesses/ to create a specific remote access"""
```
Comment (fwininger) [inline cbw_api_toolbox/cbw_api.py]: remove the / at the end of /api/v2/remote_accesses to match with the code.

File: examples/create_remote_access.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
+"""Create connexion agentless"""
```
Comment (fwininger) [inline examples/create_remote_access.py]: Create remote access

**Pull request merged**